modules/tracking/samples/lasot_benchmark.py

```diff
@@ -0,0 +1,158 @@
+import numpy as np
+import cv2 as cv
+import glob
+import argparse
+
+def init_tracker(tracker_name):
+    if tracker_name == "Boosting":
+        tracker = cv.TrackerBoosting_create()
+        frames_before_reinit = 500
+    elif tracker_name == "MIL":
+        tracker = cv.TrackerMIL_create()
+        frames_before_reinit = 1000
+    elif tracker_name == "KCF":
+        tracker = cv.TrackerKCF_create()
+        frames_before_reinit = 110
+    elif tracker_name == "MedianFlow":
+        tracker = cv.TrackerMedianFlow_create()
+        frames_before_reinit = 20
+    elif tracker_name == "GOTURN":
+        tracker = cv.TrackerGOTURN_create()
+        frames_before_reinit = 20
+    elif tracker_name == "MOSSE":
+        tracker = cv.TrackerMOSSE_create()
+        frames_before_reinit = 20
+    elif tracker_name == "CSRT":
+        tracker = cv.TrackerCSRT_create()
+        frames_before_reinit = 250
+    return [tracker, frames_before_reinit]
+
+def main():
+    parser = argparse.ArgumentParser(description="Run LaSOT-based benchmark for visual object trackers")
+    parser.add_argument("--path_to_dataset", type=str,
+                        default="Test part of LaSOT dataset", help="Full path to LaSOT folder")
```

Comment:
Please add visualization option (like `--show`) so user would be able to see how tracker works.

Reply:
Data set include 280 videos, all of them used for 7 trackers, are you sure that visualization is necessary?

Reply:
Yes, it's for debugging only. By default, should be `false`

Reply:
Added flag for visualization.